src/actions/BreachActions.js

```diff
@@ -23,18 +23,24 @@ const createConsecutiveSuccess = (sensor, updatedBreaches, updatedLogs) => ({
 });
 
 const createConsecutiveBreaches = sensor => async dispatch => {
-  const { id } = sensor;
+  const { id: sensorID } = sensor;
 
   try {
-    const logs = await BreachManager().getLogsToCheck(id);
+    const logs = await BreachManager().getLogsToCheck(sensorID);
+    const mappedToPlainObjects = logs.map(({ id, temperature, timestamp }) => ({
+      id,
+      temperature,
+      timestamp: moment(timestamp).unix(),
+    }));
     const configs = sensor.breachConfigs;
-    const mostRecentBreach = await BreachManager().getMostRecentBreach(id);
+    const mostRecentBreach = await BreachManager().getMostRecentBreach(sensorID);
     const [breaches, temperatureLogs] = await BreachManager().createBreaches(
       sensor,
-      logs,
+      mappedToPlainObjects,
       configs,
-      mostRecentBreach
+      mostRecentBreach?.toJSON()
```

Comment:
Same logic with the log above, just now when assigning an `endTimestamp` to the breach